### PR TITLE
sysmon_dns_over_https_enabled.yml

### PR DESCRIPTION
**Pull request by austinsonger**

### CODE REVIEW

File: rules/windows/registry_event/sysmon_dns_over_https_enabled.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+title: DNS-over-HTTPS Enabled by Registry
+id:04b45a8a-d11d-49e4-9acc-4a1b524407a5
```
Comment (frack113) [inline rules/windows/registry_event/sysmon_dns_over_https_enabled.yml:2]: id: 04b45a8a-d11d-49e4-9acc-4a1b524407a5

**Review by frack113 — CHANGES_REQUESTED** [review]
missing a space

File: rules/windows/registry_event/sysmon_dns_over_https_enabled.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+title: DNS-over-HTTPS Enabled by Registry
+id: 04b45a8a-d11d-49e4-9acc-4a1b524407a5
+description: Detects when a user enables DNS-over-HTTPS. This can be used to hide internet activity or be used to hide the process of exfiltrating data. With this enabled organization will lose visibility into data such as query type, response and originating IP that are used to determine bad actors.
+author: Austin Songer
+status: experimental
+references:
```
Comment (frack113) [inline rules/windows/registry_event/sysmon_dns_over_https_enabled.yml:6]: date: 2021/07/22

File: rules/windows/registry_event/sysmon_dns_over_https_enabled.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        TargetObject:
+            - 'HKEY_LOCAL_MACHINE\SOFTWARE\Google\Chrome\DnsOverHttpsMode'
+        Details: 'DWORD (secure)'
+    selection3:
+        TargetObject:
+            - 'HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Mozilla\Firefox\DNSOverHTTPS'
+        Details: 'DWORD (1)'
+    condition: selection1 or selection2 or selection3
+falsepositives:
+- "Unlikely"
```
Comment (frack113) [inline rules/windows/registry_event/sysmon_dns_over_https_enabled.yml:32]: \" not need

File: rules/windows/registry_event/sysmon_dns_over_https_enabled.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+level: "medium"
```
Comment (frack113) [inline rules/windows/registry_event/sysmon_dns_over_https_enabled.yml:33]: level: medium

**Review by frack113 — CHANGES_REQUESTED** [review]
Sorry , some more change

**Comment by Neo23x0** [comment]
Thanks

**Pull request merged**